examples/angular/tsconfig.app.json

```diff
@@ -6,5 +6,5 @@
     "types": []
   },
   "files": ["src/main.ts", "src/polyfills.ts"],
-  "include": ["src/**/*.d.ts", "../../packages/core"]
```

Comment:
"@aws-amplify/ui-core" no longer needs to be explicitly included now that it's getting built with `tsup` 👍